tests/integration/basic/test_analyze_dataset.py

```diff
@@ -47,8 +47,9 @@ def test_infer_types_on_home_rentlas(self):
         for k in type_information.identifiers:
             self.assertTrue(type_information.identifiers[k] is None)
 
-    def test_type_deduction(self):
-        n_points = 100
+    def test_with_generated(self):
+        # Must be an even number
+        n_points = 134
```

Comment:
please indicate why 134 is used as opposed to 100.

Reply:
when checking the length I wanted to make sure it's understood length == length of data. With `100` people might think it's 100% (0 to 100)